pom.xml

```diff
@@ -63,6 +63,12 @@
       <artifactId>commons-text</artifactId>
       <version>1.7</version>
     </dependency>
+    
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>2.0.5-beta</version>
+    </dependency>
```

Comment:
Note: I think @eshika used a different Mockito dependency. I don't think this is a pressing concern though, but I just wanted to note it down.